base/applications/drwtsn32/lang/es-ES.rc

```diff
@@ -0,0 +1,15 @@
+/*
+ * PROJECT:     Dr. Watson crash reporter
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     es-ES resource file
+ * COPYRIGHT:   Copyright 2018 William Kent (wjk011 [at] gmail [dot] com)
+ * TRANSLATOR:  Javier Fernández, aka "Elhoir" 2021 <elhoir AT gmail DOT com>
```

Comment:
Note: Other language files here replace the Copyright (of the file) by the translator.

Reply:
Ah, i supposed that line refers the software itself, not the locale file. Thanks for pointing.